github-actions-helpers/Build.cs

```diff
@@ -24,6 +24,9 @@ partial class Build : NukeBuild
     [Parameter("Configuration to build - Default is 'Debug' (local) or 'Release' (server)")]
     readonly Configuration Configuration = IsLocalBuild ? Configuration.Debug : Configuration.Release;
 
+    [Parameter("The current version of the source and build")]
+    readonly string Version = "0.94.0";
```

Comment:
Is this a string that must be modified during the release process? Should it be added to the doc?

Reply:
Yes I"ll add it to the release notes template. And I'll add automation for this as well in next steps